README.txt

```diff
@@ -0,0 +1,15 @@
+To install this patch: 
+
+Run these two commands from your OpenEMR installation folder. You must have composer installed and functioning on your server to run these command. 
+You can google install composer to find installation instructions for Windows and Linux.
+
+composer require twig/twig:^2.0
+
+composer require twig/extensions
+
+IF YOU INSTALL THE PATCH WITHOUT RUNNING THE TWO LINES ABOVE.
+YOU WILL BREAK YOUR INSTALL!!!!
+
+Copy the zip file to your openemr folder and unzip.
+Select overwrite when prompted.
+
```

Comment:
hi @juggernautsei , no need for this file to be included in this PR

Reply:
You are right, I have to do some changes.